cm/base/BaseCM/cm_input.py

```diff
@@ -1,8 +1,87 @@
+import logging
 import os
 
+import pyproj
+import rasterio
+from rasterstats import zonal_stats
+from shapely.geometry import shape
+from shapely.ops import cascaded_union, transform
+
+from BaseCM.cm_output import validate
+
+GEOJSON_PROJ = "EPSG:4326"
+
 
 def get_raster_path(raster_name):
     """Returns the path to the raster file based on the raster name."""
     raster_dir = os.path.join(os.environ["UPLOAD_DIR"], "raster")
     raster_path = os.path.join(raster_dir, raster_name, "raster.tiff")
     return raster_path
+
+
+class StatNotComputeError(Exception):
+    """Exception thrown for a non-existing calculation module"""
```

Comment:
The description does not seem to match the function name?

Reply:
that is correct, my bad
I will fix it